pennychat/models.py

```diff
@@ -6,19 +6,40 @@
 
 
 class PennyChat(models.Model):
-    DRAFT_STATUS = 1
-    SHARED_STATUS = 2
-    COMPLETED_STATUS = 3
+    SHARED = 20
+    COMPLETED = 30
+    ABANDONED = 40
```

Comment:
I multiplied all statuses by 10 so that we can fit things into the ranges if need be - for example, what if in the future there is more than one way to share a chat? It would be nice for that share to be `21` so that it's grouped with the existing share.

Reply:
also penny chats are created when the invitation is shared, thus penny chats don't need `DRAFT`